applications/asset_tracker_v2/src/cloud/cloud_codec/aws_iot/aws_iot_codec.c

```diff
@@ -58,14 +58,15 @@ int cloud_codec_init(struct cloud_data_cfg *cfg, cloud_codec_evt_handler_t event
 	return 0;
 }
 
-int cloud_codec_encode_neighbor_cells(struct cloud_codec_data *output,
-				      struct cloud_data_neighbor_cells *neighbor_cells)
+int cloud_codec_encode_cloud_location(
+	struct cloud_codec_data *output,
+	struct cloud_data_cloud_location *cloud_location)
```

Comment:
~Slightly confusing naming with `cloud_location`. Sort of implicates that we're trying to geolocate a cloud on the sky, but that seems like a hard problem to solve 🤠
Don't really have any better suggestions, so just keep it if others are okay with it~

Edit: I see that you already discussed this at some other time and decided to keep it

Reply:
Yep, nobody likes the name (including myself) and nobody has a better naming either :)